enviroment-files/q2-minimap2-qiime2-amplicon-2025.10.yml

```diff
@@ -11,4 +10,4 @@ dependencies:
   - q2-types
   - pip
   - pip:
-      - q2-pinocchio@git+https://github.com/bokulich-lab/q2-pinocchio.git@main
+      - git+https://github.com/bokulich-lab/q2-minimap2.git
```

Comment:
The prefix should not have been removed here - please keep the format of that string as it was and only update the names 🙏